flink-shaded-yarn-tests/pom.xml

```diff
@@ -146,42 +172,42 @@ under the License.
 							<relocations>
 								<relocation>
 									<pattern>com.google</pattern>
-									<shadedPattern>org.apache.flink.hadoop.shaded.com.google</shadedPattern>
+									<shadedPattern>org.apache.flink.hadoop2.shaded.com.google</shadedPattern>
```

Comment:
I don't think we have to modify these.

Reply:
Without this chanage, the test of `YARNHighAvailabilityITCase` cannot get success.

Reply:
I'm currently looking into why that is the case. I'd very much prefer if this module would be independent of whatever hadoop version is in flink-dist.